client/src/pages/Login.js

```diff
@@ -10,11 +10,39 @@ import SexyBike from "../assets/sexyBike.svg";
 import SexyBikeRider from "../assets/sexyBikeRider.svg";
```

Comment:
you could migrate all svg files into one, thus you ll only need to import in one line of code (../assets/assets.js)

Reply:
Will probably be resolved in another pr. 😄 

But thanks for the hint 😄